Kinvey.Core/Auth/Credential.cs

```diff
@@ -195,21 +201,35 @@ public static Credential From(User user)
 		public static Credential From(NativeCredential nc)
 		{
 			return new Credential(nc.UserID,
-			                      nc.Properties[Constants.STR_ACCESS_TOKEN],
-			                      nc.Properties[Constants.STR_AUTH_TOKEN],
-			                      nc.Properties[Constants.STR_USERNAME],
-			                      JsonConvert.DeserializeObject<Dictionary<string, JToken>>(nc.Properties[Constants.STR_ATTRIBUTES]),
-			                      JsonConvert.DeserializeObject<KinveyUserMetaData>(nc.Properties[Constants.STR_USER_KMD]),
-			                      nc.Properties[Constants.STR_REFRESH_TOKEN],
-			                      nc.Properties[Constants.STR_REDIRECT_URI],
-			                      null);
+								  nc.Properties[Constants.STR_ACCESS_TOKEN],
+								  nc.Properties[Constants.STR_AUTH_TOKEN],
+								  nc.Properties[Constants.STR_USERNAME],
+								  JsonConvert.DeserializeObject<Dictionary<string, JToken>>(nc.Properties[Constants.STR_ATTRIBUTES]),
+								  JsonConvert.DeserializeObject<KinveyUserMetaData>(nc.Properties[Constants.STR_USER_KMD]),
+								  nc.Properties[Constants.STR_REFRESH_TOKEN],
+								  nc.Properties[Constants.STR_REDIRECT_URI],
+								  null);
 		}
 
 		internal static Credential From(SQLCredential sqlcred)
 		{
 			Dictionary<string, JToken> attributes = JsonConvert.DeserializeObject<Dictionary<string, JToken>>(sqlcred.Attributes);
 			KinveyUserMetaData userKMD = JsonConvert.DeserializeObject<KinveyUserMetaData>(sqlcred.UserKMD);
-			return new Credential(sqlcred.UserID, sqlcred.AccessToken, sqlcred.AuthToken, sqlcred.UserName, attributes, userKMD, sqlcred.RefreshToken, sqlcred.RedirectUri, sqlcred.DeviceID);
+			var credential = new Credential(sqlcred.UserID, sqlcred.AccessToken, sqlcred.AuthToken, sqlcred.UserName, attributes, userKMD, sqlcred.RefreshToken, sqlcred.RedirectUri, sqlcred.DeviceID);
+			credential.SecAuthToken = sqlcred.SecAuthToken;
```

Comment:
Would not mind doing this in the future, but left it out for now because it would involve making a number of changes across numerous files that would bloat this PR.  I would prefer to tackle this in a new task, separate to this.